src/scripts/list-users.test.ts

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable functional/no-return-void */
+/* eslint-disable functional/no-expression-statement */
+/* eslint-disable functional/functional-parameters */
+import * as TE from 'fp-ts/TaskEither';
+import * as O from 'fp-ts/Option';
+import { UserService } from './services/user-service';
+import { users, usersInGroup } from './list-users';
+import { GroupService } from './services/group-service';
+
+import * as test from './services/testing-service';
+
+describe('Listing users', () => {
+  // Note that listing multiple or no users is outside the scope of the test, as that functionality comes from a dependency
+  it('passes when attempting to list users without a group without error', async () => {
+    // Given a user service that can retreive a list containing a single user
+    const userService: UserService = {
+      ...test.baseUserService(),
+      listUsers: () => TE.right([test.user]),
+    };
+
+    // When we attempt to list the users from the client
+    const result = await users(userService)();
+
+    // Then we should have a right
+    expect(result).toBeRight();
```

Comment:
You can nest your matcher in here? Don't need to pull apart the Either by hand on line 28.

Reply:
I couldn't find any methods in the jest-fp-ts API that would work as both a `toBeRight` and a `toContain`. The closest one is `toSubsetEqualRight`, which doesn't do the desired operation

Reply:
🤔 Annoying.

Reply:
Well, it seems there's not much to do about it